src/SignalRServiceExtension/TriggerBindings/ServiceContextAdapter.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Connections;
+using Microsoft.Azure.SignalR.Management;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService.TriggerBindings
```

Comment:
>Microsoft.Azure.WebJobs.Extensions.SignalRService [](start = 10, length = 49)

use `Microsoft.Azure.WebJobs.Extensions.SignalRService` so other file reference using would be clear.